### ISSUE TITLE
Simulation breaks when run with fixed date

### ISSUE THREAD

**Issue by julian-molina** [Superalgos/DesktopApp#15]
I run a simulation with the following Base Asset parameters:

```
{ 
"name": "USDT",
"initialBalance": 10,
"minimumBalance": 5,
"maximumBalance": 20000,
"initialDatetime": "2019-08-01T00:00:00.000Z"
}
```

The simulation seems to have run properly for a few candles, showing a couple of strategy entries and exits on the charts as per the image below.

![image](https://user-images.githubusercontent.com/13994516/63412679-827c4a00-c3f8-11e9-9da7-b81d76a6a857.png)

The browser's console shows the following errors, with a Jason output file not in JSON format:

![image](https://user-images.githubusercontent.com/13994516/63412835-cf602080-c3f8-11e9-8f0a-78e07da277e6.png)

This is the content of the file in question:

```[[1564661400000,1564662599999,1,0,10044.17548975,NaN,2,1]]```

If I run the simulation over the same period of time but without a fix date (placing the charts on the same start date) the simulation runs fine.

**Comment by Luis-Fernando-Molina** [comment]
> This is the content of the file in question:

> [[1564661400000,1564662599999,1,0,10044.17548975,NaN,2,1]]

Which is exactly the file in question?

**Comment by julian-molina** [comment]
This one:

![image](https://user-images.githubusercontent.com/13994516/63417368-6a5cf880-c401-11e9-97d1-b1790cc9de74.png)

**Comment by Luis-Fernando-Molina** [comment]
shere your definition, i can not reproduce it otherwise

**Comment by Luis-Fernando-Molina** [comment]
Fixed included at patch #3 to be released.
